cmd/gossamer/commands/account.go

```diff
@@ -24,15 +24,13 @@ var AccountCmd = &cobra.Command{
 	Use:   "account",
 	Short: "Create and manage node keystore accounts",
 	Long: `The account command is used to manage the gossamer keystore.
-Examples: 
+Examples:
 
-To generate a new sr25519 account: 
-	gossamer account generate --keystore-path=path/to/location --scheme=sr25519
-To generate a new ed25519 account: 
-	gossamer account generate --ed25519
```

Comment:
I took the freedom to change the existing example to use `ed25519` instead of `sr25519` since it's the default one, but if it don't make sense I just rollback and remove the ed25519 command